vars/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
 # vars file for ansible-role-openio-repository
-openio_repository_url: "http://{{ openio_repository_creds }}mirror.openio.io/pub/repo/openio"
+
+# Credentials for private repo, dependent on "with_dict" loop variable: "item"
```

Comment:
Can you explain ?

Reply:
This variable, using {{ item. }} is meant to be used in a "with_dict" loop... See its usage in the tasks files